src/pointarray.js

```diff
@@ -87,8 +87,8 @@ SVG.extend(SVG.PointArray, {
 
     // recalculate position of all points according to new size
     for (i = this.value.length - 1; i >= 0; i--) {
-      this.value[i][0] = ((this.value[i][0] - box.x) * width)  / box.width  + box.x
-      this.value[i][1] = ((this.value[i][1] - box.y) * height) / box.height + box.y
+      if(box.width) this.value[i][0] = ((this.value[i][0] - box.x) * width)  / box.width  + box.x
```

Comment:
Before it produced NaN instead so it wasnt a  noop but a bug. User should know, that if they stretch points which are on one line, nothing will happen. See #505 for the discussion on this and expected behavior.